### DISCUSSION TITLE
Plancia energia ed energia termica

### DISCUSSION THREAD

**Discussion by dante70** [indomus/forum#2997] [HUB: Home Assistant]
buongiorno,ho creato un sensore energia per monitorare energia prodotta dal pannello solare termico.
Volevo inserire il tutto nella plancia energia ed avendo libero la sezione gas ho configurato come indicato (device class ecc).
il problema e' che non viene rilevato dalla plancia( non compare nel menu a tendina per selezionarlo).
Il sensore e' funzionante come si puo' evincere dal grafico riportato
![Immagine 2024-03-24 104544](https://github.com/indomus/forum/assets/64617618/b6e902da-cfe3-42a4-a5e1-663c89d1e6cc)
![Immagine 2024-03-24 111000](https://github.com/indomus/forum/assets/64617618/36a44aa8-591d-45ab-b61f-9479b7fdd92a)

**Comment by mvimercati** [comment]
Ciao! Interessante, anche io ho un solare termico.
Ma come lo contabilizzi? Come risparmio di gas?
